client/homebrew/editor/editor.jsx

```diff
@@ -174,9 +178,42 @@ const Editor = createClass({
 		}
 	},
 
-	brewJump : function(){
-		const currentPage = this.getCurrentPage();
-		window.location.hash = `p${currentPage}`;
+	brewJump : function(targetPage=this.getCurrentPage()){
+		if(!window || this.isMeta()) return;
+		// console.log(`Scroll to: p${targetPage}`);
+		window.frames['BrewRenderer'].contentDocument.getElementById(`p${targetPage}`).scrollIntoView({ behavior: 'smooth', block: 'start' });
+		// const hashPage = (page != 1) ? `p${page}` : '';
+		// window.location.hash = hashPage;
+	},
+
+	sourceJump : function(targetLine=null){
+		if(this.isText() || (this.isStyle() && targetLine == 0)) {
```

Comment:
Same thing, maybe `left` and `right` should not appear (or be greyed out) in the Style tab instead of checking here.

Reply:
SnippetBar buttons removed; Divider buttons only appear in Content Editor.